samples/client/petstore/kotlin-gson/src/main/kotlin/org/openapitools/client/apis/PetApi.kt

```diff
@@ -75,6 +72,24 @@ class PetApi(basePath: kotlin.String = defaultBasePath) : ApiClient(basePath) {
         }
     }
 
+    /**
+    * Add a new pet to the store
+    * 
+    * @param body Pet object that needs to be added to the store 
+    * @return ApiInfrastructureResponse<Unit?>
+    * @throws UnsupportedOperationException If the API returns an informational or redirection response
+    * @throws ClientException If the API returns a client error response
+    * @throws ServerException If the API returns a server error response
+    */
+    @Throws(UnsupportedOperationException::class, ClientException::class, ServerException::class)
+    fun addPetWithHttpInfo(body: Pet) : ApiInfrastructureResponse<Unit?> {
```

Comment:
I'm not also sure about the name `withHttpInfo`

Reply:
`Detailed`? I used `WithHttpInfo` since that's what it was in the past, but I'm fine with anything :)

Reply:
Please keep WithHttpInfo for the time being as that's what we've been using in other clients (e.g. C#, Ruby, etc)